src/JWT.ts

```diff
@@ -363,15 +389,17 @@ export async function verifyJWT(
   const skewTime = typeof options.skewTime !== 'undefined' && options.skewTime >= 0 ? options.skewTime : NBF_SKEW
   if (signer) {
     const nowSkewed = now + skewTime
```

Comment:
I think you misunderstood what I meant by the `now` parameter.
I was referring to an override to the internal now, which is currently computed as `const now: number = Math.floor(Date.now() / 1000)`

My proposal was that the policy can contain a `now` override as a `number`, not a `boolean`.
This number would be used instead of `new Date().now()/1000` and should represent a timestamp.

Reply:
Sorry for misunderstanding, fix it and replaced the test case for it as well. Thank you!